src/components/select/select.js

```diff
@@ -116,6 +116,9 @@ angular.module('material.components.select', [
 function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $parse) {
   return {
     restrict: 'E',
+    scope: {
+      selectedText: '=?selectedText'
```

Comment:
Ahh yes, this is why the test broke, I have now changed the code to copy the same methods that are used for `mdOnClose` / `mdOnOpen`